internal/pkg/cli/job_init.go

```diff
@@ -323,6 +323,32 @@ func (o *initJobOpts) askSchedule() error {
 	return nil
 }
 
+func (o *initJobOpts) legitimizePlatform() error {
```

Comment:
We don't necessarily need to address this in this PR. If this logic is the same for both job and service, what do you think of this function signature
```
func legitimizePlatform(image string, dockerEngine dockerEngine) (platform *string, error)
```
and call it as
```
platform, err := legitimizePlatform(o.image, o.dockerEngine)
if err != nil {
   return err
}
o.platform = platform
```
?

Reply:
I like that. I need to a do a larger refactor that eliminates a lot of the redundancy among the workloads, so I'll include this in that refactor!